python/dolma/cli/deduper.py

```diff
@@ -121,6 +121,9 @@ class DeduperConfig:
         default=False,
         help="If true, only print the configuration and exit without running the deduper.",
     )
+    mounted: bool = field(
```

Comment:
Minor: if `mounted` is primarily used to avoid creating temp files, maybe we can rename it to something like `not_local_temp_files` And have the help message explain that behavior (it can even still say "useful for when documents are on an S3 mounted directory").